node-scripts/src/utils/model.js

```diff
@@ -68,12 +68,19 @@ class DockerComposeMavenDeployment {
   }
 }
 class DockerComposeGenericMavenDeployment {
-  constructor(mavenProject, services) {
+  constructor(mavenProject, services, mavenUrl) {
     this.mavenProject = mavenProject;
     this.services = services;
     this.mavenUrl = this.mavenUrl;
   }
 }
+
+class DockerComposeGenericDeployment {
```

Comment:
Add some JS doc describing the type.
```
A type which assumes that the Docker Compose project will be installed through the processing of the `artifacts` section.
```

Reply:
@enyachoke , did you drop this in your latest commit?